src/assisted_test_infra/test_infra/helper_classes/config/base_terraform_config.py

```diff
@@ -19,7 +17,9 @@ class BaseTerraformConfig(BaseNodesConfig, ABC):
     single_node_ip: str = None
     dns_records: Dict[str, str] = field(default_factory=dict)
 
-    libvirt_uri: str = consts.DEFAULT_LIBVIRT_URI
+    remote_shell_address: str = None
+    remote_shell_private_key: str = None
```

Comment:
you don't need that, right? terraform will reach the remote machine through the libvirt uri

Reply:
oh sorry, it's the config, not the parameters passed to terraform :facepalm: